engine/common/requester/engine.go

```diff
@@ -382,15 +383,20 @@ func (e *Engine) dispatchRequest() (bool, error) {
 	}()
 
 	e.metrics.MessageSent(e.channel.String(), metrics.MessageEntityRequest)
+	e.log.Debug().
+		Str("sender_node_id", e.me.NodeID().String()).
```

Comment:
we probably don't need to print our own node_id. it's included in messages by default

Reply:
https://github.com/onflow/flow-go/pull/3270/commits/1b7cdf8fad7f2603c66d5e96f613834dcad24e78